### PR TITLE
show large thumbnail preview when hovering on search result thumbnail #10310

### PR DESCRIPTION
**Pull request by chrabyrd**
### Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Issues Solved

#10310

### CODE REVIEW

**Review by chiatt — CHANGES_REQUESTED** [review]
This looks really good, but I noticed that for landscape images it would be nice to perhaps have a light gray background or maybe just a border so that the title doesn't appear pushed to the right by whitespace. 
image

**Comment by chrabyrd** [comment]
> This looks really good, but I noticed that for landscape images it would be nice to perhaps have a light gray background or maybe just a border so that the title doesn't appear pushed to the right by whitespace. image

@chiatt resolved in latest

File: arches/app/templates/views/components/search/search-results.htm [hunk 1]
```diff
@@ -8,6 +8,23 @@
...
+                        event: { 
+                            mouseover: mouseoverThumbnail,
+                            mouseout: mouseoutThumbnail,
+                        }
+                    "
+                >
+                    <img style="max-width: 70px;" data-bind="attr:{src: thumbnail}">
+                </div>
+
+                <img style="position: absolute; left: 380px; display: none;" data-bind="attr:{src: thumbnail}">
```
Comment (chiatt) [inline arches/app/templates/views/components/search/search-results.htm:25]: Absolute positioning with left/right is a problem for RTL. Is there another way to do this?

**Review by chiatt — CHANGES_REQUESTED** [review]

File: arches/app/templates/views/components/search/search-results.htm (hunk 1 above)
Comment (chrabyrd) [reply arches/app/templates/views/components/search/search-results.htm]: fixed in latest

**Review by chiatt — APPROVED** [review]

**Pull request merged**